src/app.js

```diff
@@ -406,6 +406,10 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   const framingService = new FramingService(sidePanel, menuicons, mainview, this._components.resizeFeature)
   framingService.start()
 
+  // preactivate solidity plugin
```

Comment:
I think we should preactivate solidity and udapp, only if the `workspace` is empty, not always

Reply:
your are right.  I'll do the change.